drivers/pinctrl/Kconfig.mcux

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2021 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config PINCTRL_MCUX_RT
+	bool "Pin controller driver for MCUX RT1xxx MCUs"
+	depends on SOC_SERIES_IMX_RT
+	default y
```

Comment:
nit: default based on `dt_compat_status`, see other drivers

Reply:
Fixed in latest push